### PR TITLE
Add ISSUE_TEMPLATE-s to avoid lost bug reports

### PR DESCRIPTION
**Pull request by kgyrtkirk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/proposed-feature.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Proposed Feature
+about: Propose a solution to a problem or wishlist item
+title: ''
+labels: proposed-feature
+assignees: ''
```
Comment (mkindahl) [inline .github/ISSUE_TEMPLATE/proposed-feature.md:6]: Not sure what the difference between this and a feature request is.

**Review by mkindahl — APPROVED** [review]
LGMT

File: .github/ISSUE_TEMPLATE/proposed-feature.md (hunk 1 above)
Comment (fabriziomello) [reply .github/ISSUE_TEMPLATE/proposed-feature.md:6]: I agree with @mkindahl ... can we simplify it and have just bug report and feature request template??

Comment (kgyrtkirk) [reply .github/ISSUE_TEMPLATE/proposed-feature.md:6]: agree; I've copied the templates from `timescaledb-toolkit` and kept which seemed relevant.

I've removed this template

**Review by mkindahl — APPROVED** [review]

**Pull request merged**